source/jormungandr/jormungandr/interfaces/v1/Schedules.py

```diff
@@ -203,6 +234,11 @@ def get(self, uri=None, region=None, lon=None, lat=None):
             self.region = i_manager.get_region(region, lon, lat)
         timezone.set_request_timezone(self.region)
 
+        # create direction type filter
+        if args['direction_type'] and args['direction_type'] != 'all':
```

Comment:
I would find it more straightforward to check `'all'` case inside `_add_direction_type_filter`, as for now you manage this case in 2 parts of the code (this check prevents bug from previous comment I think).

Reply:
I understand your point of you and i coded like that in first step. 
I have choosen that to highlight the fact of "skip proces"s when it is `all` case.

Reply:
An other reason is when you don't need to add another filter option, we want to avoid to call the add_direction_type_filter function. It avoids, inside, to re-copy the filter into the output of the function. We avoid a useless copy. You see what I mean ?